src/components/FinishedPrompt.jsx

```diff
@@ -62,15 +62,22 @@ class FinishedPrompt extends React.Component {
       )
         .filter(task => isAQuestionTask(task, this.props.workflow))
       : [];
+    const language = this.props.currentLanguage;
+    const translations = (this.props.translatedWorkflow && this.props.translatedWorkflow[language])
+      ? this.props.translatedWorkflow[language] : null;
+    const currentTask = tasks && tasks.length && tasks[0].taskId;
+    const instructionsTranslation = translations && translations[`tasks.${currentTask}.help`];
+    const headerTranslation = translations && translations[`tasks.${currentTask}.question`];
+
+    const instructions = instructionsTranslation || translate('finished.instructions') + translate('finished.whenReady');
+    const header = headerTranslation || translate('finished.allTranscribed');
```

Comment:
[MINOR] This makes perfect sense to me, but I'm wondering how easy it'll be for future devs to understand that we're using Pandora translations and then falling back to local translations.